lms/djangoapps/verify_student/tests/test_models.py

```diff
@@ -504,6 +505,28 @@ def test_verification_for_datetime(self):
         result = SoftwareSecurePhotoVerification.verification_for_datetime(deadline, query)
         self.assertEqual(result, second_attempt)
 
+    @ddt.unpack
+    @ddt.data(
+        {'enrollment_mode': 'honor', 'status': (None, None), 'output': 'N/A'},
+        {'enrollment_mode': 'verified', 'status': (False, False), 'output': 'Not ID Verified'},
+        {'enrollment_mode': 'verified', 'status': (True, True), 'output': 'ID Verified'},
+        {'enrollment_mode': 'verified', 'status': (True, False), 'output': 'ID Verification Expired'}
+    )
+    def test_verification_status_for_user(self, enrollment_mode, status, output):
+        """
+        Verify that verification_status_for_user works.
+        """
+        user = UserFactory.create()
+        course = CourseFactory.create()
+
+        with patch('verify_student.models.SoftwareSecurePhotoVerification.user_is_verified') as mock_verification:
+            with patch('verify_student.models.SoftwareSecurePhotoVerification.user_is_reverified_for_all') as mock_re_verification:  # pylint: disable=line-too-long
```

Comment:
Instead of adding a pylint exception you can create a variable for the path to the method:
```
user_is_reverified_for_all_path = 'verify_student.models.SoftwareSecurePhotoVerification.user_is_reverified_for_all'
```